addons/mail/models/res_config_settings.py

```diff
@@ -36,8 +36,12 @@ class ResConfigSettings(models.TransientModel):
         'Twilio Account Auth Token',
         config_parameter='mail.twilio_account_token',
     )
-    primary_color = fields.Char(related='company_id.primary_color', string="Header Color", readonly=False)
-    secondary_color = fields.Char(related='company_id.secondary_color', string="Button Color", readonly=False)
+    email_primary_color = fields.Char(
+        related='company_id.email_primary_color', string="Header Color",
```

Comment:
Strings are not aligned, is it on purpose? 

If we want to make it shorter in the form I would re-define the string in the xml and keep the more meaningful one here.